applications/CompressiblePotentialFlowApplication/tests/naca0012_small_test/naca0012_small_parameters.json

```diff
@@ -130,6 +138,19 @@
                 "tolerance"             : 1e-9,
                 "relative_tolerance"    : 1e-9
             }
+        },{
+            "python_module"   : "compare_two_files_check_process",
+            "kratos_module"   : "KratosMultiphysics",
+            "help"            : "",
+            "process_name"    : "CompareTwoFilesCheckProcess",
+            "Parameters" :{
+                "output_file_name"    : "cl_points_with_lift.dat",
+                "reference_file_name" : "cl_ref.dat",
```

Comment:
aren't you missing the reference file?

Reply:
it is comparing the value obtained by cl_points_with_lift to the reference file which is under it; and then deleting it.
No errors are shown